src/libs/ajax.js

```diff
@@ -919,6 +919,11 @@ const Workspaces = signal => ({
 
 
 const DataRepo = signal => ({
+  getMetadata: async () => {
+    const res = await fetchDataRepo(`repository/v1/search/metadata`, _.merge(authOpts(), { signal }))
```

Comment:
I think our convention is single quotes unless interpolating. I use the opposite convention in my Ruby code, so I'm open to always using backticks for strings.
```suggestion
    const res = await fetchDataRepo('repository/v1/search/metadata', _.merge(authOpts(), { signal }))
```